client/state/selectors/get-current-plan-purchase-id.js

```diff
@@ -21,7 +21,7 @@ import { getCurrentPlan } from 'state/sites/plans/selectors';
  * @returns {?number}        Purchase ID if known
  */
 export default function getCurrentPlanPurchaseId( state, siteId ) {
-	const result = get( getCurrentPlan( state, siteId ), 'id', null );
```

Comment:
This might be an issue: the original `get()` call was configured to return `null` on a failure. `get( foo, 'bar.baz', null )` is equivalent to `foo?.bar?.baz ?? null`